docs/src/workflows/daemon.md

```
@@ -37,6 +37,19 @@ keyhog daemon status
 keyhog daemon stop
 ```
 
+### Background guard daemon lifecycle
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)